src/Systems/MultiplayerSystem.cpp

```diff
@@ -285,7 +296,7 @@ void MultiplayerSystem::update(const float deltaTime)
             break;
         case SynchronisedEvent::UpdateType::MOVEMENT:
             movementEvents.push_back(eventEntity);
-        default:
+        default: {}
```

Comment:
Nie trzeba tak robić

Reply:
Aha, @dominiks01 mi to zaznaczył to zmieniłem na jego prośbę. Jest o to temat w konwersacji.